lib/drivers/dexcomDriver.js

```diff
@@ -1003,6 +1008,8 @@ module.exports = function (config) {
                 progress(100);
                 cb(err, data);
               });
+            } else {
+              cb(err, data);
```

Comment:
Was the lack of this the regression re: the error bubbling up to be caught by the UI?